Library/Homebrew/utils/github.rb

```diff
@@ -342,8 +342,8 @@ def merge_pull_request(repo, number:, sha:, merge_method:, commit_message: nil)
     open_api(url, data: data, request_method: :PUT, scopes: CREATE_ISSUE_FORK_OR_PR_SCOPES)
   end
 
-  def print_pull_requests_matching(query)
-    open_or_closed_prs = search_issues(query, type: "pr", user: "Homebrew")
+  def print_pull_requests_matching(query, only)
```

Comment:
```suggestion
  def print_pull_requests_matching(query, only: nil)
```

Reply:
`def print_pull_requests_matching(query, only: nil)` caused error like below.
```
$brew search ruby --pul-request
Error: wrong number of arguments (given 2, expected 1)
Please report this issue:
  https://docs.brew.sh/Troubleshooting
/usr/local/Homebrew/Library/Homebrew/utils/github.rb:345:in `print_pull_requests_matching'
/usr/local/Homebrew/Library/Homebrew/cmd/search.rb:106:in `search'
/usr/local/Homebrew/Library/Homebrew/brew.rb:124:in `<main>'
```

I will write `def print_pull_requests_matching(query, only = nil)`

Reply:
> `def print_pull_requests_matching(query, only: nil)` caused error like below.

Yes, all the callers will need updated too. I think my version is a bit more readable.